src/kernel-install/90-loaderentry.install.in

```diff
@@ -126,6 +126,7 @@ elif [ -f /usr/lib/kernel/devicetree ]; then
 fi
 if [ -n "$DEVICETREE" ]; then
     for prefix in \
+        "$KERNEL_INSTALL_CONF_ROOT" \
```

Comment:
if the variable is not set this would mean we'd end up looking in the root dir, no?

Reply:
Oops.